app/decorators/service_provider_session_decorator.rb

```diff
@@ -68,7 +68,7 @@ def sp_return_url
   end
 
   def cancel_link_path
-    sign_up_start_path(request_id: sp_session[:request_id])
+    sign_up_start_path(request_id: sp_session[:request_id], locale: locale_url_param)
```

Comment:
Should we be doing this everywhere we're using path helpers, or is this a special case?

Reply:
We may want to do this everywhere we're using path helpers? This is overridden with a default in the controller context... (see `ApplicationController#default_url_options`) so that means everything not inside a template or a controller we could be missing

Reply:
Cool, def something to look out moving forward